rules/xcodeproj.bzl

```diff
@@ -542,8 +542,19 @@ def _populate_xcodeproj_targets_and_schemes(ctx, targets, src_dot_dots, all_tran
         target_settings["SWIFT_ACTIVE_COMPILATION_CONDITIONS"] = " ".join(
             ["\"%s\"" % d for d in defines_without_equal_sign],
         )
+
+        # Without `strip` here the paths won't be created
+        # correctly in the LLLDB init file
+        fmw_search_paths_lldb = target_settings["FRAMEWORK_SEARCH_PATHS"].strip("\"").split(" ")
```

Comment:
insteaof reading this out of the settings and having to split the string, should we get the raw array of the search paths?

Reply:
Yep, as I was bringing back the quoted paths I thought of the same thing. Working on exactly his now 👍